src/main/java/com/dougnoel/sentinel/webdrivers/WebDriverFactory.java

```diff
@@ -92,32 +94,57 @@ public static WebDriver instantiateWebDriver() {
         String downloadDirectory = ConfigurationManager.getOptionalProperty("download");
         if (downloadDirectory != null)
             DownloadManager.setDownloadDirectory(downloadDirectory);
+        
+        String runOnGrid = ConfigurationManager.getOptionalProperty("runOnGrid");
+    	String browser = ConfigurationManager.getBrowserName();
+    	
+        if (runOnGrid != null && runOnGrid.equals("true")) {
+        	String gridUrl = ConfigurationManager.getOptionalProperty("gridUrl");
+        	String browserVersion = ConfigurationManager.getOptionalProperty("browserVersion");
+        	
+        	//use full browser version from grid in yml or command line e.g: 88.0.4324.96
+        	driver = instantiateRemoteWebDriver(browser, browserVersion, gridUrl);
```

Comment:
Is there functionality for the  passing of `latest` or something similar in grid so that we can have a default?

Reply:
few things we will do in next sprint: 
1. when we will implement webdrivermanager, then we will handle latest browser version at the same time
2. will work with tony(devops) to use short browser versions e.g 88, 89, 90 etc
3. will think about it how to interpret latest when there will be multiple browser versions

Reply:
Ok, make a ticket to keep track of this work now then I'll resolve this comment.